R/sharepoint_client.R

```diff
@@ -0,0 +1,136 @@
+#' Create sharepoint client, to manage HTTP requests to sharepoint.
+#'
+#' @keywords internal
```

Comment:
add `@noRd` too for all the internal things?

Reply:
Fixed - thanks